src/screens/ContactScreen/elements/Label.js

```diff
@@ -6,7 +6,7 @@ const Label = ({
   label,
   containerBgColor = COLORS.concrete,
   titleHeaderColor = COLORS.whiteSmoke,
-  textColor = COLORS.emperor
+  textColor = COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.emperor` to `COLORS.white2` for the `textColor` prop seems fine as long as it aligns with the new color scheme you're implementing. However, ensure that this change doesn't affect readability due to contrast issues with the background.
```diff
-  textColor = COLORS.emperor
+  textColor = COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->